README.md

```diff
@@ -52,6 +52,8 @@ Migrations prefixed with `unsafe_` will warn when invoked. The API is designed t
 
 Migrations prefixed with `safe_` prefer concurrent operations where available, set low lock timeouts where appropriate, and decompose operations into multiple safe steps.
 
+Default behavior of this gem [disables ddl transactions](./lib/pg_ha_migrations.rb:8). You can change this by resetting `ActiveRecord::Migration.disable_ddl_transactions` in your application.
```

Comment:
Maybe add as a prefix to this paragraph:

> Because [running multiple DDL statements inside a transaction...acquire[s] exclusive locks on all of [the modified] objects](https://medium.com/braintree-product-technology/postgresql-at-scale-database-schema-changes-without-downtime-20d3749ed680#cc22), we default to [disabling DDL transactions...]

Reply:
`disable_ddl_transactions` should be singular, I believe.